src/EventHandler.ts

```diff
@@ -20,11 +20,13 @@ export class Event {
         this.callbacks.push(callback);
     }
 
-    executeCallback(params?: any | any[]) {
+    executeCallback(params: any[]) {
```

Comment:
Thanks @yeexinc for helping to review. @fanwgwg, if this requires documentation change please also update `ReadMe.md` and other affected files.